PCIe 6.0 Packet Generator and Exerciser/Google Test/Google Test.vcxproj

```diff
@@ -33,9 +33,11 @@
   <ImportGroup Label="PropertySheets" />
   <PropertyGroup Label="UserMacros" />
   <ItemGroup>
+    <ClInclude Include="..\PCIe 6.0 Packet Generator and Exerciser\layers\globals.h" />
```

Comment:
Why are the globals files added to the Google Test project? Please remove them

Reply:
done